### DISCUSSION TITLE
How do I access my ISP's management network? (TG799vac Xtream / VANT-W, Telia)

### DISCUSSION THREAD

**Discussion by s90961624** [hack-technicolor/hack-technicolor#319] [Q&A]
Hello!

I was able to root my router first try following the wiki and video on crazy power switching. Thanks for all the good information collected here.

I had to downgrade from v18.3 to a rootable v17.2.0339, I also upgraded to v17.2.0405 while maintaining root. I now want to dedicate some resources url-sniffing to find download link for my previous v18.3. However, trying to reach any of the posted links in the [firmware repository](https://hack-technicolor.readthedocs.io/en/stable/Repository/#tg799vac-xtream-vant-w_1) times out!

I am SSH:ed as root to my router and running `wget`, is there something I need to change in my configs before I can access the management network? See the below snippet.

```sh
root@OpenWrt:~# wget -T 15 http://131.116.22.230/172405w1441030closed.rbi
Connecting to 131.116.22.230 (131.116.22.230:80)
wget: download timed out
root@OpenWrt:~# 
```

Below are some snippets from `/etc/config/network` that may be of interest (I have not changed anything, this just peaked my interest after running `grep -rn "mgmt" -C 1 /etc/config/`).
```
config interface 'mgmt'
        option proto 'dhcp'
        option initboot '1'
        option ip4table 'mgmt'
        option reqopts '1 3 6 15 33 42 43 51 121 249'
        option release '0'
        option iface6rd '0'
        option dnsset 'mgmt'
        option auto '0'

config device 'vlan_mgmt'
        option type '8021q'
        option ifname 'eth4'
        option name 'vlan_mgmt'
        option vid '294'
        option macaddr '<redacted>'
        option mtu '1500'
```

**Comment by LuKePicci** [comment]
Just make sure the mgmt interface is up and has an IP assigned. Then it is sufficient to wget by using that IP as source.

**Reply by s90961624** [reply]
I am not sure what I need to do. I also do not see the interface at in `ip link show`.
```sh
root@OpenWrt:~# ifup mgmt
root@OpenWrt:~# ifstatus mgmt
{
        "up": false,
        "pending": false,
        "available": false,
        "autostart": true,
        "dynamic": false,
        "proto": "dhcp",
        "device": "mgmt",
        "data": {

        },
        "errors": [
                {
                        "subsystem": "interface",
                        "code": "NO_DEVICE"
                }
        ]
}
root@OpenWrt:~# ip link show | grep mgmt
19: atm_mgmt: <BROADCAST,MULTICAST> mtu 1500 qdisc qos-tch state DOWN mode DEFAULT group default qlen 1000
root@OpenWrt:~#
```

**Comment by NI-Pro** [comment]
@s90961624 Did you manage to obtain the 18.3 firmware file for your router? I have found 18.3.0757-1441012 for VBNT-H not VANT-W. But I can try to search for the VANT-W URL if you want
